### DISCUSSION TITLE
blog/how-to-find-out-docker-ip-on-ubuntu

### DISCUSSION THREAD

**Discussion by giscus** [trevortylerlee/astro-micro#52] [Announcements]
# blog/how-to-find-out-docker-ip-on-ubuntu

how to easily retrieve the IP address of a running Docker container on Ubuntu using a simple Docker command.

http://localhost:4321/blog/how-to-find-out-docker-ip-on-ubuntu

**Comment by fajarspace** [comment]
Test komen
